### PR TITLE
Update craft to 3.2

### PR DESCRIPTION
**Pull request by boxadesign**

### CODE REVIEW

**Comment by boxadesign** [comment]
@kierenbeckett Yes. 3.2 has a couple of new features that we need.

- Entries bin - this allows us to properly poll for deleted entries
- Headless preview - allows us to use the live preview functionality with our generator.

Is the problem the fact that it's a beta release or updating in general?

**Comment by kierenbeckett** [comment]
It's mainly that we wouldn't be using a released version of craftcms, but essentially whatever happens to be on master right now (semi-tested?).  Do you know when 3.2 is planned for release?

**Comment by boxadesign** [comment]
@kierenbeckett It's scheduled for tomorrow.

File: src/composer.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
     "docs": "https://craftcms.com/docs",
     "rss": "https://craftcms.com/changelog.rss"
   },
-  "prefer-stable": true,
+  "prefer-stable": false,
```
Comment (kierenbeckett) [inline src/composer.json]: Not sure what process changed this, but feels like it would ideally stay on true?

**Pull request merged**